### PR TITLE
Login Screen

### PR DESCRIPTION
**Pull request by dhanraj1925**
## Added
- added login screen design

### CODE REVIEW

File: src/screens/login/index.js [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+          backAction,
+        );
+    
+        return () => backHandler.remove();
+      }, []);
+    
+    
+    return(
+        <ScrollView style={styles.container}>
+            <StatusBar backgroundColor={'#fff'} barStyle={'dark-content'} />
```
Comment (kaushallalani) [inline src/screens/login/index.js:40]: Color aaya static che aa change kar vanu che.

File: src/screens/login/index.js
Comment (kaushallalani) [inline src/screens/login/index.js:1]: aa file full static che aa change karvani che k nai ???

**Review by kaushallalani — CHANGES_REQUESTED** [review]

**Review by kaushallalani — APPROVED** [review]

File: src/screens/login/index.js [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+            <View  style={styles.header}>
+                <View style={{justifyContent: 'space-between', flexDirection: 'row', alignItems: 'center'}}>
+                    <View style={{flex: 1}}/>
+                    <Image source={icon} style={{width: wp(35), height: wp(35)}} resizeMode='contain' />
+                    <View style={{flex: 1}}/>
+                </View>
+                <View style={{alignSelf: 'center', alignItems: 'center', width: '80%'}}>
+                    <Text style={{fontFamily: 'Satoshi-Bold', fontSize: 30, color: '#000', marginVertical: wp(5)}}>Sign In</Text>
```
Comment (Riya0212) [inline src/screens/login/index.js:48]: direct text component use karjo

File: src/screens/login/index.js [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+                        <View style={{flex: 1, height: 1, backgroundColor: '#B0B0B0'}} />
+                    </View>
+                    <View style={styles.iconContainer}>
+                        <Image source={google} style={{width: wp(8), height: hp(6)}} resizeMode='contain'/>
+                        <Image source={apple} style={{width: wp(8), height: hp(6)}} resizeMode='contain'/>
+                    </View>
+                    <View style={{flexDirection: 'row', alignItems: 'center', marginTop: hp(5)}}>
+                        <Text style={{fontFamily: 'Satoshi-Medium', color: '#383838'}}>Not A Member? </Text>
+                        <TouchableOpacity onPress={()=>{navigation.navigate(NAVIGATION.register)}} >
+                            <Text style={{fontFamily: 'Satoshi-Medium', color: '#288CE9'}}>Register Now</Text>
```
Comment (Riya0212) [inline src/screens/login/index.js:95]: bring strings from localisation

**Review by Riya0212 — CHANGES_REQUESTED** [review]

**Review by Riya0212 — APPROVED** [review]

**Pull request merged**